lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -244,6 +245,14 @@ class ClickDebouncer {
     } else if (event.type == 'pointerdown') {
       _startDebouncing(event, data);
     } else {
+      if (event.type == 'pointerup') {
+        // Record the last pointerup event even if not debouncing. This is
+        // because the sequence of pointerdown-pointerup could indicate a
+        // long-press, and the debounce timer is not long enough to capture it.
+        // If a "click" is observed after a long-press it should be
+        // debounced.
```

Comment:
I suggest "discarded" to make it extra clear what you mean here.
```suggestion
        // discarded.
```